pages/mes-jeunes/envoie-message-groupe.tsx

```diff
@@ -0,0 +1,149 @@
+import { AppHead } from 'components/AppHead'
+import Button, { ButtonStyle } from 'components/ui/Button'
+import { Jeune } from 'interfaces/jeune'
+import { GetServerSideProps } from 'next'
+import Link from 'next/link'
+import { useState } from 'react'
+import styles from 'styles/components/Layouts.module.css'
+import useMatomo from 'utils/analytics/useMatomo'
+import { Container } from 'utils/injectionDependances'
+import { withMandatorySessionOrRedirect } from 'utils/withMandatorySessionOrRedirect'
+import BackIcon from '../../assets/icons/arrow_back.svg'
+import Etape1Icon from '../../assets/icons/etape_1.svg'
+import Etape2Icon from '../../assets/icons/etape_2.svg'
+import SendIcon from '../../assets/icons/send.svg'
+
+type EnvoieMessageGroupe = {
+  jeunes: Jeune[]
+}
+
+function EnvoieMessageGroupe({ jeunes }: EnvoieMessageGroupe) {
+  const [selectedJeunes, setSelectedJeunes] = useState<Jeune[]>([jeunes[0]])
+  const [message, setMessage] = useState<string>('')
+
+  const FormIsValid = () => message !== '' && selectedJeunes.length !== 0
+
+  useMatomo('Message - Rédaction')
+
+  return (
+    <>
+      <AppHead titre='Envoie de message à plusieurs jeunes' />
+      <div className={`flex items-center ${styles.header}`}>
+        <Link href={'/mes-jeunes'}>
+          <a className='items-center mr-4'>
+            <BackIcon role='img' focusable='false' aria-hidden={true} />
+          </a>
+        </Link>
+        <h1 className='text-l-medium text-bleu_nuit'>
+          Envoi d’un message à plusieurs jeunes
+        </h1>
+      </div>
+      <div className={styles.content}>
+        <form method='POST' role='form' onSubmit={() => {}} onReset={() => {}}>
+          <div className='text-sm-regular text-bleu_nuit mb-8'>
+            Les champs marqués d&apos;une * sont obligatoires.
```

Comment:
```suggestion
            Tous les champs sont obligatoires
```
on peut le laisser en visible pour les lecteurs d'écran et permettre d'identifier en amont que tous les champs sont obligatoires.